subprojects/plugins/src/integTest/groovy/org/gradle/java/compile/AbstractJavaGroovyCompileAvoidanceIntegrationSpec.groovy

```diff
@@ -293,39 +276,7 @@ public class ToolImpl {
                 public Object s = "12";
                 public void execute() { String s = toString(); }
             }
-"""
-
-        then:
-        succeeds ":b:${language.compileTaskName}"
-        executedAndNotSkipped ":a:${language.compileTaskName}"
-        skipped ":b:${language.compileTaskName}"
-
-        when:
-        // add static initializer and constructor
-        sourceFile.text = """
-            public class ToolImpl {
-                static { }
-                public ToolImpl() { }
-                public Object s = "12";
-                public void execute() { String s = toString(); }
-            }
-"""
-
-        then:
-        succeeds ":b:${language.compileTaskName}"
-        executedAndNotSkipped ":a:${language.compileTaskName}"
-        skipped ":b:${language.compileTaskName}"
-
-        when:
-        // change static initializer and constructor
```

Comment:
Does changing an already existing static initializer and constructor affect Groovy compilation? I would expect that the task stays up-to-date as well. We could add this test to the base class as a separate test. Or do we have that coverage already?

Reply:
Ok, sure, separated